common:
    lid_topic:  "/livox/lidar"
    imu_topic:  "/Aliengo/imu"
    time_sync_en: false         # ONLY turn on when external time synchronization is really not possible

preprocess:
    lidar_type: 1                # 1 for Livox serials LiDAR, 2 for Velodyne LiDAR, 3 for ouster LiDAR, 
    scan_line: 6
    blind: 0

mapping:
    acc_cov: 0.1
    gyr_cov: 0.1
    b_acc_cov: 0.0001
    b_gyr_cov: 0.0001
    fov_degree:    90
    det_range:     450.0
    extrinsic_est_en:  false      # true: enable the online estimation of IMU-LiDAR extrinsic
    extrinsic_T: [ 0.0, 0.0, 0.0 ]
    extrinsic_R: [ 1, 0, 0,
                   0, 1, 0,
                   0, 0, 1]

publish:
    path_en:  true
    scan_publish_en:  true       # false: close all the point cloud output
    dense_publish_en: true       # false: low down the points number in a global-frame point clouds scan.
    scan_bodyframe_pub_en: true  # true: output the point cloud scans in IMU-body-frame

pcd_save:
    pcd_save_en: true
    interval: -1                 # how many LiDAR frames saved in each pcd file; 
                                 # -1 : all frames will be saved in ONE pcd file, may lead to memory crash when having too much frames.

GoToXYTheta:
    Max_ETF: 0.1745
    TOL_FINDIST: 0.1
    DIST_DA: 0.3
    HIST_ETF: 0.1745
    DIST_NEWPOSE: 0.5
    THETA_DA: 0.43625
    TOL_FINTHETA: 0.18725
    THETA_NEWPOSE: 0.45
    VEL_ANG_NOM: 0.25
    VEL_LIN_NOM: 0.3
    GAIN_FWD: 1
    VEL_LIN_DA: 0.1
    GAIN_DA: 0.5
    VEL_ANG_DA: 0.1
